tests/kas_with_instance.spec.ts

```diff
@@ -183,3 +183,89 @@ test('create and delete a Kafka Topic', async ({ page }) => {
   await createKafkaTopic(page, testTopicName);
   await deleteKafkaTopic(page, testTopicName);
 });
+
+// test_4kafka.py test_edit_topic_properties_after_creation
+test('edit topic properties after creation', async ({ page }) => {
+  await navigateToKafkaTopicsList(page, testInstanceName);
+  await createKafkaTopic(page, testTopicName);
+
+  const row = page.locator('tr', { hasText: testTopicName });
+  await row.locator('[aria-label="Actions"]').click();
+  await page.getByText('Edit topic configuration').click();
+
+  await expect(page.locator('input[name="num-partitions"]')).toHaveValue('1', { timeout: 3000 });
+  const numPartitionsBefore: string = await page.locator('input[name="num-partitions"]').getAttribute('value');
+  console.log('numPartitionsBefore: ' + numPartitionsBefore);
```

Comment:
maybe you could use a more meaning full message?

Reply:
what do you suggest @Frawless ?

Reply:
added `console.log('Number of partitions by default: ' + numPartitionsBefore);`

Reply:
Can I resolve this conversation @Frawless ?